packages/spright-components/src/accordion/index.ts

```diff
@@ -0,0 +1,27 @@
+import {
```

Comment:
I'm slightly on the fence about including the accordion example. It hasn't gone through all the required steps to be in spright but because it's a real useful component I could see people starting to use it. 

If we need a component to prove our infrastructure is working I wonder if we should do something to make it obvious that this is a placeholder. Ideas:
1. give it an obvious flaw like incorrect styling (hot pink?)
2. give it a name that conveys its nature ("temporary-accordion"?)
3. replace it with a component that's obviously useless ("spright-rectangle").